package/gluon-core/files/lib/gluon/upgrade/010-primary-mac

```diff
@@ -28,6 +28,16 @@ elseif platform.match('ar71xx', 'generic', {'unifi-outdoor-plus', 'carambola2'})
   table.insert(try_files, 1, '/sys/class/net/eth0/address')
 elseif platform.match('ar71xx', 'generic', {'archer-c5', 'archer-c7'}) then
   table.insert(try_files, 1, '/sys/class/net/eth1/address')
+elseif platform.match('ar71xx', 'generic', {'mr600', 'mr600v2',
+                                            'mr900', 'mr900v2',
+                                            'mr1750',
+                                            'om2p',
+                                            'om2pv2', 'om2p-hs', 'om2p-hsv2', 'om2p-hsv3',
+                                            'om2p-lc',
+                                            'om5p',
+                                            'om5p-an',
+                                            'om5p-ac', 'om5p-acv2'}) then
+  table.insert(try_files, 1, '/sys/class/net/eth0/address')
```

Comment:
There is already an elseif branch for eth0, please change that one instead of adding a new one.